COH_Australia/Australia_importer.py

```diff
@@ -0,0 +1,578 @@
+        """
+        if descriptions:
+            self.wd.add_multiple_descriptions(descriptions, item)
+
+    def commit_claims(self, protoclaims, item, default_ref):
+        """
+        Add each claim (if new) and source it.
+
+        :param protoclaims: a dict of claims with
+            key: Prop number
+            val: Statement|list of Statements
+        :param item: the target entity
+        :param default_ref: main/default reference to use
+        """
+        for prop, statements in protoclaims.items():
+            if statements:
+                statements = helpers.listify(statements)
+                statements = set(statements)  # eliminate potential duplicates
+                for statement in statements:
+                    # check if None or a Statement(None)
+                    if (statement is not None) and (not statement.isNone()):
+                        # use internal reference if present, else the general
+                        ref = statement.ref or default_ref
+                        self.wd.addNewClaim(
+                            prop, statement, item, ref)
+
+                        # reload item so that next call is aware of changes
+                        item = self.wd.QtoItemPage(item.title())
+                        item.exists()
+
+    def make_protoclaims(self, data):
+        """
+        Construct potential claims for an entry.
+
+        :param data: dict of data for a single heritage object
+        """
+        protoclaims = dict()
+
+        # P17: country
+        protoclaims['P17'] = WdS.Statement(self.country)
+
+        # P1435: heritage status
+        heritage_type = self.get_heritage_type(data.get('type'))
+        statement = WdS.Statement(self.status[heritage_type])
+        if data.get('register_date'):
+            statement.addQualifier(
+                WdS.Qualifier('P580', self.parse_date(
+                    data.get('register_date'))))
+        protoclaims['P1435'] = statement
+
+        # P31: class
+        protoclaims['P31'] = WdS.Statement(
+            self.instance_type[data.get('class').lower()])
+
+        # P3008: place_id
+        protoclaims[self.place_id_p] = WdS.Statement(data['place_id'])
+
+        # P131: state
+        protoclaims['P131'] = WdS.Statement(
+            self.get_state(data['state'], data['address']))
+
+        # P2046: area
+        if data.get('hectares'):
+            protoclaims['P2046'] = WdS.Statement(
+                pywikibot.WbQuantity(
+                    data['hectares'], unit=self.hectares, site=self.wd.repo))
+
+        # P969: address
+        if ',' in data['address']:
+            protoclaims['P969'] = WdS.Statement(data['address'])
+
+        # P276: place
+        # format for address is "street, town STATE_ISO"
+        # might be possible to match town to a wikidata entry?
+        # but how to ensure the correct state?
+        pywikibot.warning('You never implemented P276')
```

Comment:
But there are P276 in the preview?

Reply:
Looks like my last commit was never pushed. Will make sure it comes with the review follow-up